kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonWindowController.java

```diff
@@ -172,6 +191,14 @@ private void update() {
                 buf.setFormat(SkeletonSettings.FORMAT_TYPE.COMPACT);
             }
 
+            /*
+             * 
+             * TODO: Discuss, if this is the correct way to obtain the FxController.
+             * As of now, the code to extract the controller class name would exist on
+             * 3 different locations.
```

Comment:
This approach looks ok to me

Reply:
Thanks for review. I will remove the TODO.